server/msteams/client.go

```diff
@@ -1949,7 +1949,7 @@ func (tc *ClientImpl) ListChannels(teamID string) ([]clientmodels.Channel, error
 	return channels, nil
 }
 
-func (tc *ClientImpl) ListChannelMessages(teamID string, channelID string, since time.Time) ([]*clientmodels.Message, error) {
+func (tc *ClientImpl) OnChannelMessagesSince(teamID string, channelID string, since time.Time, callback func(*clientmodels.Message)) error {
```

Comment:
The method name of `OnChannelMessagesSince` doesn't parse for me -- it sounds like I'm registering a callback handler for some event. Would it make sense to keep `ListChannelMessages` but accept a `chan *clientmodels.Message` instead?

Reply:
We can build it in that way, but I found it simple, the other approach would need to have 2 goroutines collaborating and synchronizing, this way it is more straight forward, IMO, but I'm ok moving that to a channel, not a big deal.